src/core/Tabs/Tabs.tsx

```diff
@@ -57,6 +57,10 @@ export type TabsProps = {
    * Custom CSS class name for tab panel.
    */
   contentClassName?: string;
+  /**
+   * Custom CSS class name for the tabs wrapper.
+   */
+  wrapperClassName?: string;
```

Comment:
Hm, will users will understand what "wrapper" stand for. What if we name this prop "containerClassName"?

Reply:
wrapper and container kinda mean different things though. container implies that it contains multiple instances of something. That's why we have `TagContainer`